renderer/components/UI/CopyButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import copy from 'copy-to-clipboard'
+import Button from 'components/UI/Button'
+import Copy from 'components/Icon/Copy'
+
+const CopyButton = ({ value, hint, onCopy, p, ...rest }) => {
+  const doCopy = () => {
+    copy(value)
+    if (onCopy) {
+      onCopy(value)
+    }
+  }
+
+  const paddingProps = p
+    ? {
+        px: p,
+        py: p,
+      }
+    : null
```

Comment:
shouldn't it just be `p` ?

Reply:
Ideally, but `px` and `py` override `p` and the `Button` element already has `px` and `py` set so passing `p` would not take effect.